### ISSUE TITLE
UI Improvements | Refactoring

### ISSUE THREAD

**Issue by ghost** [MNoya/DotaCraft#209]
- [x] Refactor lua-side UI functionality, localise in ui.lua
- [x] Reformat panaroma script files into seperate sub-directories based on specific functionality
- [x] Add new implementation images to precache layout
- [x] Localise panel sorting function inside GameUI.
- [ ] Review previous UI modules
  - [ ] Hero Panels
  - [ ] Control Groups
  - [ ] Trading Alliances
  - [ ] Team Selection (still needs some fixes)
  - [ ] End Screen (still needs to be completed in styling)
  - [ ] Unit Shops
- [ ] Double check previous additions to confirm they did not break in the process of refactoring

**Comment by ghost** [comment]
> Reformat panaroma files into seperate sub-directories based on specific functionality

Can still be expanded further but most of the scripts left are single-script functionalities.

**Comment by ghost** [comment]
@MNoya 
Move unit_shops to ui sub-directory?

There's also some ui functionality that might be getting used im not sure where it's located I will most likely dig through it tomorrow, unless you want to handle those later when you're available (or point out which files I should look in)

Any Criticism, hit me i'll adjust it accordingly when I review this again tomorrow.

**Comment by MNoya** [comment]
yeah shops subdirectories sounds good, I think stuff that has 2 or more files deserves its own folder. All this changes are looking good :smiley:

**Comment by ghost** [comment]
UI refactoring coming along nicely, currently what's left to do is to review the ui modules i've implemented and refactor those. There's also the possibility of refactoring more in the future, possibly more UI localisation for lua based scripts, some UI is still getting used from non-ui subdirectory components(in lua).

**Comment by ghost** [comment]
Expanded UI revision list

**Comment by MNoya** [comment]
Refactoring shops now
